transactions/sell_item.cdc

```diff
@@ -3,11 +3,13 @@ import NonFungibleToken from "../contracts/NonFungibleToken.cdc"
 import FlowToken from "../contracts/FlowToken.cdc"
 import ExampleNFT from "../contracts/ExampleNFT.cdc"
 import NFTStorefront from "../contracts/NFTStorefront.cdc"
+import MetadataViews from "../contracts/MetadataViews.cdc"
```

Comment:
where can i read the code for `MetadataViews.cdc` ?

Reply:
https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc